transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
```

Comment:
Shall I modify it to "Representation of a DNS record as expected by the API" then?